tests/test_netcdf_reader.py

```diff
@@ -35,11 +35,6 @@ def test_large_grids_are_thinned(big_xdata, big_ydata):
     assert len(result) == (constants.DEFAULT_SPATIAL_AXIS_SIZE * 4) - 3
 
 
-def test_small_grids_are_not_thinned(xdata, ydata):
-    result = netcdf_reader.thinned_perimeter(xdata, ydata)
```

Comment:
Consideration of small grids was abandoned with the switch to using shapely LineString interpolation to figure out the points along each side of the perimeter.